scripts/run_vox.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 module load stopos
 
-voxdir=${PBS_O_HOME}/EC14-voxelyze/voxelyzeMain
+voxdir=${PBS_O_HOME}/thesis/voxelyze/voxelyzeMain
```

Comment:
Woops, thats what the folder is named on my machine. All the EC14 feels so bloated. I'll look into how to get this to be more generic.